packages/bpk-component-label/src/BpkLabel.js

```diff
@@ -31,10 +31,19 @@ export type Props = {
   disabled: boolean,
   required: boolean,
   white: boolean,
+  invalid: boolean,
```

Comment:
A label cannot have a concept of valid/invalid.
The field-set should be handling setting the colour as it's not a concern of the label.

The red colour used should also be darker to maintain good contrast.

Reply:
@georgegillams How can a label have a concept of `required`, or `disabled`, both present in the props, but not a concept of `invalid`?

Reply:
Now moved to the fieldset